Sources/SwiftDocC/Model/Rendering/DocumentationContentRenderer.swift

```diff
@@ -134,6 +134,10 @@ public class DocumentationContentRenderer {
         case .collectionGroup: return role(for: nodeKind)
         default: break
         }
+
+        if let pageKind = article.metadata?.pageKind {
```

Comment:
Should we put this above line 133 as well? Or does it make sense in that case to ignore the metadata directive?

Reply:
To be honest i'm not 100% sure what situations the `collectionGroup` node kind is used for internally. But going from the comment, it seems safe to move the metadata check above it, since the directive is guaranteed to be what the author intends for a given page.